master/usage/configuration/bgp.md

```diff
@@ -148,12 +148,12 @@ To remove a global BGP peer that was configured with IP address 192.20.30.40
 (the AS number is not required when deleting) run the following command on any
 node:
 
-	$ calicoctl delete bgpPeer 192.20.30.40 --scope=global
+	$ calicoctl delete bgpPeer 192.20.30.40 
```

Comment:
with v3, you delete the bgppeer using it's name (not the IP), so it'll be `calicoctl delete bgppeer bgppeer-global-3040`